sdk/state/update.go

```diff
@@ -0,0 +1,143 @@
+package state
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/stellar/experimental-payment-channels/sdk/txbuild"
+	"github.com/stellar/go/xdr"
+)
+
+type PaymentProposal struct {
+	IterationNumber            int64
+	ObservationPeriodTime      time.Duration
+	ObservationPeriodLedgerGap int64
```

Comment:
I think you are probably on to something here, with capturing these values in the payment and we'll probably need this at some point down the line, but I think we can assume the values will be constants until we look at adding support for changing the observation period during the lifetime of the channel. Would you be okay removing these two for the moment?

Reply:
I wasn't 100% if these should be passed from Channel to payments, in the case the Channel changes these values before close, payments before and after would store the different values which could be useful. Removing for now for simplicity sgtm though
https://github.com/stellar/experimental-payment-channels/pull/55/commits/69cb66636c808d80826fde64e5ea4f68999bcbfb